templates/terraform/custom_expand/firewall_log_config.go.erb

```diff
@@ -0,0 +1,32 @@
+<%- # the license inside this block applies to this file
+	# Copyright 2020 Google Inc.
+	# Licensed under the Apache License, Version 2.0 (the "License");
+	# you may not use this file except in compliance with the License.
+	# You may obtain a copy of the License at
+	#
+	#     http://www.apache.org/licenses/LICENSE-2.0
+	#
+	# Unless required by applicable law or agreed to in writing, software
+	# distributed under the License is distributed on an "AS IS" BASIS,
+	# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	# See the License for the specific language governing permissions and
+	# limitations under the License.
+-%>
+func expand<%= prefix -%><%= titlelize_property(property) -%>(v interface{}, d TerraformResourceData, config *Config) (interface{}, error) {
+	l := v.([]interface{})
+	transformed := make(map[string]interface{})
+	if len(l) == 0 || l[0] == nil {
+		// send enable = false to ensure logging is disabled if there is no config
+		transformed["enable"] = false
```

Comment:
we should check if `enable_logging` is set here too, otherwise we will overwrite the field

Reply:
ah, yes, i was writing this as if i had already removed it, rather than just deprecating it. thanks!